### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by dayou5168**
Page not displaying properly. fix this.

### CODE REVIEW

File: docs/reference/README.md [hunk 1]
```diff
@@ -217,6 +217,7 @@ For running a node cluster of load balanced Lotus JSON RPC API endpoints.
 - [Hactar](https://www.hactar.app/) - analytics for your Filecoin miner node.
 - [Lotus Farcaster](https://github.com/s0nik42/lotus-farcaster) - Prometheus, Grafana and Python monitoring dashboard.
 - [Mining Benchmarks](https://filecoin-benchmarks.on.fleek.co/): Hardware performance comparisons from the Filecoin community.
+- [Sectors repair](https://github.com/froghub-io/filecoin-sealer-recover) Open source sectors repair tool
```
Comment (johnnymatthews) [inline docs/reference/README.md:220]: I'm not familiar with this project. Can you explain what it's about, and why it should be in this list?

Comment (dayou5168) [reply docs/reference/README.md:220]: Hi. Johny.  here is the detail why i add this item:  ​it's an open-source tool for repairing sectors that are damaged or lost. i talked to the author and he said he will maintain the update.  i think this tool can help our storage provide avoiding loss. and I already summit a pr to that repo which translate the `README.md` to English.
of course, this is optional tool.   if you like it too. just add it . if not . we can delete this line.

Comment (johnnymatthews) [reply docs/reference/README.md:220]: Ok, that makes sense. Thanks for the explanation. For now, let's leave this out of this PR. We should get a member of the Filecoin dev team to take a look at the froghub-io/filecoin-sealer-recover repo before adding it into these docs.

Comment (johnnymatthews) [reply docs/reference/README.md:220]:
```suggestion
```

Comment (johnnymatthews) [reply docs/reference/README.md:220]: This rest of this PR is good to go!

**Review by johnnymatthews — APPROVED** [review]

**Comment by johnnymatthews** [comment]
Just realized that "Chain management" is no longer in this repo, but in https://github.com/filecoin-project/lotus-docs/. PR here: https://github.com/filecoin-project/lotus-docs/pull/50. 

Closing this PR.

**Pull request closed without merging**